from .cache import Cache, CustomLockError
